drivers/storage/portworx/component/csi.go

```diff
@@ -1035,7 +1038,7 @@ func (c *csi) createCSIDriver(
 				Name: csiConfig.DriverName,
 			},
 			Spec: storagev1.CSIDriverSpec{
-				AttachRequired:       boolPtr(false),
+				AttachRequired:       boolPtr(true),
```

Comment:
Can this always be `true` or is only needed for windows driver?
cc: @ggriffiths

Reply:
This will result in the CSI driver requiring attachment objects for both windows and linux, which is not needed.

There's no way to add this for windows only, as there's one CSIDriver object in a cluster for Portworx.

Reply:
This might not be required.